portal/gil/templates/gil/base.html

```diff
@@ -387,13 +387,14 @@ <h4 class="box-modal__title">{{_("Session Timed Out")}}</h4><p class="box-modal_
                     onwarning: function() {$("#session-warning-modal").modal("show"); if (sessMon.modalShown) sessMon.intervalMonitor = setInterval(function(){ sessMon.ontimeout() }, 2 * 60 * 1000);}
               });
               window.sessMon = sessMon;
-               // Configure the session timeout warning modal
+              var warningText = i18next.t("Your session will expire in approximately {time} seconds due to inactivity.".replace("{time}",(sessMon.timeBeforeWarning / 1000)));
```

Comment:
Whoops, almost missed that- can you make sure `replace()` is called on/after `t()`? (It looks like `replace()` is being called before `t()`)